content/bluetooth.md

```diff
@@ -70,6 +71,19 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
 
 ## Bluetooth Troubleshooting
 
+<!--## Dear Pop OS team I hope this helps to improve your distribution and documentation, feel free to rework.-->
```

Comment:
Please remove this comment. Discussion about the PR belongs in the discussion thread on Github, not in comments in the code.